### ISSUE TITLE
Clicking on sender or people in cc where only mail is given uses mail as name too

### ISSUE THREAD

**Issue by jancborchardt** [owncloud/mail#1168]
Alex sends an email to Christoph and me. Now he only put in our email addresses without our names.

When I click on Christophs email address to send him a message, the email is duplicated and used as name as well:

> "christoph@example.at" <christoph@example.at>

cc @owncloud/mail

**Comment by irgendwie** [comment]
What is the expected result here? Just the e-mail without a name?

**Comment by jancborchardt** [comment]
Yep. Only `christoph@example.at` without any apostrophes or brackets.

**Comment by ChristophWurst** [comment]
I guess this comes from the horde mail address class. If I remember correctly, it uses the mail address as label if there is no label.
However, this should be fixed on the client side. Other code parts (e.g. displaying message recipients) rely on that functionality.
